app/main.py

```diff
@@ -2,24 +2,60 @@
 
 
 class IntegerRange:
-    pass
+    def __init__(self, min_value: int, max_value: int) -> None:
+        self.min_value = min_value
+        self.max_value = max_value
+
+    def __set_name__(self, instance: object, name: str) -> None:
+        self.private_name = f"_{name}"
+
+    def __get__(self, instance: object, owner: object) -> str:
+        return getattr(instance, self.private_name)
+
+    def __set__(self, instance: object, value: int) -> None:
+        if not isinstance(value, int):
+            raise TypeError(f"{self.private_name} must be integer!")
+        if not self.min_value <= value <= self.max_value:
+            raise ValueError(f"Value must be in range from "
+                             f"{self.min_value} to {self.max_value}")
+        setattr(instance, self.private_name, value)
 
 
 class Visitor:
-   pass
+    def __init__(self, name: str, age: int, weight: int, height: int) -> None:
+        self.name = name
+        self.age = age
+        self.weight = weight
+        self.height = height
```

Comment:
The `Visitor` class is missing data validation. You should validate the input data in the constructor (`__init__`) method. You can use the `IntegerRange` class for this purpose. This way, you can ensure that the age, weight, and height of the visitor are always within a valid range.

Reply:
This data is checked in IntegerRange